developer/bake_config.py

```diff
@@ -50,20 +50,12 @@
 BUNDLE_CACHE_ROOT_FOLDER_NAME = "bundle_cache"
 
 
-def _should_skip_caching_sparse(desc):
+def _should_skip(types, desc):
     """
-    Returns if a descriptor's content should not be cached.
-
-    We should not attempt to cache descriptors that are path-based. Not only don't they
-    need to be cached, but they might be using special tokens like CONFIG_FOLDER
-    that can't be understood outside a pipeline configuration. We also skip caching
-    app_store descriptors in sparse configs since SG Desktop will take care of downloading
-    these automatically from the app store at runtime.
-
-    :param dict desc: Descriptor to check.
+    Check if a descriptor is for the appstore so it can be skipped.
```

Comment:
Is this true? The `types` is passed so we don't know what descriptor type will be in the list.